BenchmarkVCFs/VcfdistBenchmark.wdl

```diff
@@ -0,0 +1,72 @@
+version 1.0
+
+workflow VcfdistEval {
+    input {
+        File truth_vcf
+        File eval_vcf
+        File bed_file
+        File fasta_file
+        String docker = "us.gcr.io/broad-dsde-methods/vcfdist:v0.1"
+        Int? verbosity
+    }
+
+    call RunVcfdistTask {
+        input:
+            truth_vcf = truth_vcf,
+            eval_vcf = eval_vcf,
+            bed_file = bed_file,
+            fasta_file = fasta_file,
+            docker = docker,
+            verbosity = select_first([verbosity, 0])
```

Comment:
I would instead write in the inputs above `Int verbosity = 0` to mean it's optional as an input, but defaults to `0`.

Reply:
(in the end I changed the default to 1, as that's what vcfdist uses as the default)